content/en/llm_observability/monitoring/cost_tracking.md

```diff
@@ -0,0 +1,67 @@
+- Correlate LLM cost with performance, errors, prompt content, and application behavior in a single view
+
+<div class="alert alert-info"> If you provide only partial cost information, for example, supplying a total cost but not input/output cost, Datadog will still compute the input and output cost using provider pricing. This may cause a mismatch between your manually provided total cost and the sum of Datadog’s computed input/output costs. Datadog will always display your provided total cost as-is, even if these values differ.</div>
+
+## Set up Cost Tracking
+Datadog supports two ways of enabling Cost Tracking: Automatic and Manual. Most users rely on automatic cost calculation, which Datadog automatically estimates the cost by:
+- Using the token counts returned by the provider (or computed from the response), and
+- Applying the provider’s published pricing for the specific model
+
+You can also supply your own cost data using the manual option. In this case, Datadog will use your values directly and will not attempt to automatically calculate costs.
+
+### Automatic
+If your application uses one of the supported providers listed below [link], Datadog will automatically collect token counts and calculate the cost of each request.
+
+### Manual
+For custom LLM providers, self-hosted models, or unsupported frameworks, you can manually provide cost values. To manually override or supply cost information, follow the instrumentation steps described here: [Link to SDK reference page]
+
+<div class="alert alert-info">If you provide only partial cost information, for example, supplying a total cost but not input/output token details, Datadog will still compute the input and output token cost using provider pricing. This may cause a mismatch between your manually provided total cost and the sum of Datadog’s computed input/output costs. Datadog will always display your provided total cost as-is, even if these values differ.</div>
+
+## Supported Providers
+Datadog automatically calculates the cost of LLM requests made to the following supported providers using the publicly available pricing information from their official documentation.
+
+<div class="alert alert-info">Cost tracking currently supports text-based models only.</div>
+
+- OpenAI: [OpenAI Pricing][1]
+- Anthropic: [Claude Pricing][2]
+- Azure OpenAI: [Azure OpenAI Pricing][3]
+    - We use US East 2 and Global Standard Deployment Pricing for all requests
+- Gemini: [Gemini Pricing][4]
+
+## Metrics
+Cost Tracking metrics can be found in [LLM Observability Metrics][5].
+
+## View Cost Tracking in LLM Observability
+{{< img src="llm_observability/cost_tracking_overview.png" alt="Cost view for an app in LLM Observability." style="width:100%;" >}}
+
+Once cost tracking is enabled, open your application in LLM Observability and select Cost on the left sidebar. The Cost view provides a complete, visual breakdown view of your token usage and LLM spend:
```

Comment:
Let's copy the format of the prompts page, and keep this paragraph quick and brief